corehq/apps/smsbillables/async_handlers.py

```diff
@@ -102,7 +102,9 @@ def _fmt_success(self, response):
 
 class PublicSMSRatesAsyncHandler(BaseAsyncHandler):
     slug = 'public_sms_rate_calc'
-    allowed_actions = 'public_rate'
+    allowed_actions = [
+        'public_rate',
+    ]
```

Comment:
yeah seems like this was wrong. seems like we got away with it because of how the [validation](https://github.com/dimagi/commcare-hq/blob/3dc00f772650c7fca26b2be074123179e38ab1f9/corehq/apps/hqwebapp/async_handler.py#L73) is done